<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>631</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
   <property name="currentIndex">
    <number>6</number>
   </property>
   <widget class="QWidget" name="home_page">
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>340</y>
       <width>491</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="reg_pill">
        <property name="text">
         <string>약 등록</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="user_conti">
        <property name="text">
         <string>사용자 내역</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="user_manag">
        <property name="text">
         <string>사용자 관리</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>home</string>
     </property>
    </widget>
    <widget class="QLabel" name="home_img">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>491</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="reg_page">
    <widget class="QPushButton" name="reg_to_home">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>사용자별 약 등록</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>340</y>
       <width>481</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="reg_save">
        <property name="text">
         <string>저장</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reg_update">
        <property name="text">
         <string>수정</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>551</width>
       <height>35</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QComboBox" name="reg_choose_pill">
          <item>
           <property name="text">
            <string>약 선택</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reg_addpill_btn">
          <property name="text">
           <string>약 추가</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>약 선택</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="reg_select_user">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>사용자 선택</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이명운</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>박세린</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>남상기</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="reg_backbtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>뒤로가기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reg_nextbtn">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>310</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>다음</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>591</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_2">
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>551</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QComboBox" name="reg_choose_time">
          <item>
           <property name="text">
            <string>복용시간 선택</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>요일 선택</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>복용 시간 선택</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="reg_day_of_the_week">
          <item>
           <property name="text">
            <string>요일선택</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>월</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>화</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>수</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>목</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>금</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>토</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>일</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>개수 선택</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="count_pill"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3"/>
    </widget>
   </widget>
   <widget class="QWidget" name="conti_page">
    <widget class="QComboBox" name="conti_select_user">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>사용자 선택</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이명운</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>박세린</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>남상기</string>
      </property>
     </item>
    </widget>
    <widget class="QTableWidget" name="conti_user_status">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>361</width>
       <height>192</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>복용 시간</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>복용 상태</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>사용자 내역</string>
     </property>
    </widget>
    <widget class="QPushButton" name="conti_to_home">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="manag_page">
    <widget class="QComboBox" name="manag_select_user">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>사용자 선택</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>이명운</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>박세린</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>남상기</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="manag_to_home">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>120</y>
       <width>160</width>
       <height>150</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="manag_delete_btn">
        <property name="text">
         <string>삭제</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="manag_modi">
        <property name="text">
         <string>수정</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fake_btn">
        <property name="text">
         <string>더보기..</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>사용자 관리</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_modify">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>431</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="title">
      <string>modify</string>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>411</width>
        <height>224</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>연락처</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextEdit" name="modi_phone">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QTextEdit" name="modi_address">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QTextEdit" name="modi_disease">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>주소</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>사용자 병명 수정</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="modi_back_btn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>뒤로가기</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modi_complete_btn">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>280</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>수정 완료</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_delete">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>431</width>
       <height>341</height>
      </rect>
     </property>
     <property name="title">
      <string>delete</string>
     </property>
     <widget class="QPushButton" name="manag_delete_Y">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>152</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>예</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>139</x>
        <y>95</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>삭제 하시겠습니까 ?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="manag_delete_N">
      <property name="geometry">
       <rect>
        <x>152</x>
        <y>182</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>아니요</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QTextEdit" name="say">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>341</width>
       <height>70</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>약 이름을 입력하세요 </string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>190</y>
       <width>91</width>
       <height>70</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="add_cancel">
        <property name="text">
         <string>취소</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_save">
        <property name="text">
         <string>저장</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>461</width>
       <height>321</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>카드등록을 해서 로그인해주세요.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="move_reg_card">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>410</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>카드 등록</string>
     </property>
    </widget>
    <widget class="QPushButton" name="move_del_card">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>410</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>삭제</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>601</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>성별</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QTextEdit" name="receive4_address"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>연락처</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTextEdit" name="receive1_name"/>
      </item>
      <item row="4" column="1">
       <widget class="QTextEdit" name="receive5_uid"/>
      </item>
      <item row="2" column="1">
       <widget class="QTextEdit" name="receive3_phone"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>UID</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>주소</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="receive2_sex">
        <item>
         <property name="text">
          <string>성별을 선택하세요</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>F</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>이름</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="reg_admin_cancel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>420</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>취소</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reg_admin_save">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>420</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>등록 완료</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>등록 화면</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
